app.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express');
+const morgan = require('morgan');
+
+const app = express();
+
+app.use(express.json());
+app.use(morgan('dev'));
+
+app.get('/users', async (req, res) => {
```

Comment:
use the express router to define a new file named `routs/userRouter.js` and define all routes there